src/DiffOp/Diff/Diff.php

```diff
@@ -383,7 +385,7 @@ private function getNewOffset(): int {
 	 *
 	 * @param mixed $value
 	 */
-	public function append( $value ) {
+	public function append( $value ): void {
```

Comment:
Is this related?

Reply:
Yes, in PHP 8.1 deprecation warning will be thrown 
`Deprecated: Return type of Diff\DiffOp\Diff\Diff::append($value) should either be compatible with ArrayObject::append(mixed $value): void, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in /Users/milic/Development/Diff/src/DiffOp/Diff/Diff.php on line 395`

Reply:
Wouldn't using `#[\ReturnTypeWillChange]` make more sense for the time being then? At least, until we can drop support for older PHP versions?

Reply:
👍